packages/integration-tests/src/entities/Tag.factories.ts

```diff
@@ -1,7 +1,8 @@
-import { DeepNew, EntityManager, FactoryOpts, newTestInstance } from "joist-orm";
+import { DeepNew, FactoryOpts, newTestInstance } from "joist-orm";
+import type { EntityManager } from "./entities";
 import { Tag } from "./entities";
 
-// Example of using a completely different opts type. Use number so that the type isn't a string/look like a tagged id.
+/** Example of using a completely different opts type. Use number so that the type isn't a string/look like a tagged id. */
```

Comment:
NOTE: I explicitly didn't ignore this document, to illustrate that you can provide helpful comments for these if you enhance the default factories.